src/test/groovy/com/bisnode/opa/client/OpaClientBuilderSpec.groovy

```diff
@@ -0,0 +1,54 @@
+package com.bisnode.opa.client
+
+
+import com.bisnode.opa.client.query.QueryForDocumentRequest
+import com.bisnode.opa.client.rest.ContentType
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.github.tomakehurst.wiremock.WireMockServer
+import spock.lang.Shared
+import spock.lang.Specification
+
+import static com.bisnode.opa.client.rest.ContentType.Values.APPLICATION_JSON
+import static com.github.tomakehurst.wiremock.client.WireMock.*
+
+class OpaClientBuilderSpec extends Specification {
+
+    private static int PORT = 8181
+    private static String url = "http://localhost:$PORT"
+
+    @Shared
+    private WireMockServer wireMockServer = new WireMockServer(PORT)
+
+    def setupSpec() {
+        wireMockServer.start()
+    }
+
+    def cleanupSpec() {
+        wireMockServer.stop()
+    }
+
+    def 'configure OpaClient with custom ObjectMapper'() {
```

Comment:
I think it would be nice to have also test for not setting objectMapper, or setting it to null. (I guess it should be switched to default ObjectMapper)